src/ngraph/runtime/plaidml/plaidml_compiler.cpp

```diff
@@ -87,7 +88,7 @@ std::shared_ptr<ngraph::runtime::plaidml::PlaidML_Executable>
     pass_manager.set_per_pass_validation(false);
 
     // We apply the same general-purposes passes as the CPU backend.
-    pass_manager.register_pass<ngraph::pass::FusedOpDecomposition>();
+    pass_manager.register_pass<ngraph::pass::FusedOpDecomposition>([] (const Node& node) -> bool {if (node.description() == "GroupConvolution") return true; return false;});
```

Comment:
I imagine this will fail the style check (along with a few other things throughout the PR), so run `make style-apply` if CI complains. Outside the clang-format rules, nGraph coding standards require single-statement `if` bodies to be enclosed in curly braces.